tests/python/unittest/test_meta_schedule_schedule_rule_mlt_tc.py

```diff
@@ -211,7 +212,7 @@ def matmul_relu_0(A: T.Buffer((128, 128), "float16"), B: T.Buffer((128, 128), "f
     actual = generate_design_space(
         kind="cuda",
         mod=mod,
-        target=tvm.target.Target("cuda"),
+        target=tvm.target.Target("cuda --arch=sm_50"),
```

Comment:
It's so confusing that we mark arch `sm_50` for tensor core unit tests. I don't think these changes are related to the `compute_inline` enhancement

Reply:
Let me explain: without an sm version that doesn’t support async pipeline, those unittests will fail due to software pipelining. In fact, all those tests will fail on my machine locally

Reply:
In this case, I recommend changing to `sm_70` or `sm_75`

Reply:
Will do